src/cadence/testnet/getNFTs.cdc

```diff
@@ -2131,4 +2133,40 @@ pub fun getRacingTimeNFT(owner: PublicAccount, id: UInt64): NFTData? {
             "ipfs": nft!.data!.ipfs
         }
     )
+}
+
+// https://flow-view-source.com/testnet/account/0x9f2eb43b6df02730/contract/Momentables
+pub fun getMomentables(owner: PublicAccount, id: UInt64): NFTData? {
+
+    let contract = NFTContractData(
+        name: "Momentables",
+        address: 0x9f2eb43b6df02730,
+        storage_path: "Momentables.CollectionStoragePath",
+        public_path: "Momentables.CollectionPublicPath",
+        public_collection_name: "Momentables.MomentablesCollectionPublic",
+        external_domain: "https://nextdecentrum.com"
+    )
+
+    let col = owner.getCapability(Momentables.CollectionPublicPath)
+        .borrow<&{Momentables.MomentablesCollectionPublic}>()
+
+    if col == nil { return nil }
+
+    let nft = col!.borrowMomentables(id: id)
+    if nft == nil { return nil }
+
+    //let metadata = Gaia.getTemplateMetaData(templateID: nft!.data.templateID)
+    let ipfsURL = "https://gateway.pinata.cloud/ipfs/".concat(nft!.imageCID);
+
+    return NFTData(
+        contract: contract,
+        id: nft!.id,
+        uuid: nft!.uuid,
+        title: nft!.name,
+        description: nft!.description,
+        external_domain_view_url: nil,
+        token_uri: nil,
+        media: [NFTMedia(uri: ipfsURL, mimetype: "image")],
+        metadata: nft!.getTraits(),
```

Comment:
Thanks for submitting this PR! We just updated the repo to store metadata as a {String: String?} dictionary, could you pull from the latest main branch and merge?

Reply:
Looks great otherwise!

Reply:
Sure Thanks. Will check it out.

Reply:
Hi @niveda-krish We have a Nested metadata structure as shared below. Not sure why you moved from String: AnyStruct to String: String. I am trying a few other ways to concatenate those nested objects but not sure if that might be rendered correctly. Do you have any suggestions are is there a way Alchemy can support AnyStruct ?

`	{"Hallowed Accessory": {
		"rarity": "12.69%",
		"primaryPower": "Wisdom 3d4",
		"additionalData": "Darkness",
		"name": "Gilded Eye",
		"secondaryPower": "Constitution 2d6"
	},
	"Kingdom": {
		"name": "The Seventh Kingdom",
		"rarity": "15.22%"
	}
}`

Please let me know.

Reply:
For now, I did some parsing of the above data to match Alchemy's metadata structure. However, I still believe the AnyStruct could have been a better option. Please suggest.

Reply:
@niveda-krish  Updated the PR with the new metadata structure. Kindly review and merge

Reply:
Hey @palani-kochatech, thanks for updating the contract. We are working with the Flow team on a new metadata standard. Until we have that in place, we expect to support only {String : String?} metadata to have consistency across all contracts. One way to go about serializing AnyStruct data in your case would be to iterate and store all the nested key-value pairs in the metadata struct!

Reply:
Your updates look great! Could you resolve merge conflicts with latest main branch?